primitives/transaction/src/historic_transaction.rs

```diff
@@ -56,6 +56,7 @@ impl HistoricTransaction {
         block_time: u64,
         transactions: Vec<ExecutedTransaction>,
         inherents: Vec<Inherent>,
+        equivocation_locator: Vec<EquivocationLocator>,
```

Comment:
```suggestion
        equivocation_locator: Vec<EquivocationProof>,
```
All call sites have `EquivocationProof`s, not equivocation locators. Maybe it'd be better to get it as such, makes the call sites easier.